lib/templates/rails/scaffold_controller/controller.rb

```diff
@@ -0,0 +1,97 @@
+# rubocop:disable Lint/Syntax
```

Comment:
RuboCop tries to parse this file as plain ruby instead of erb so it fails. I tried renaming to .erb but then the Rails generator doesn't pick it up.